webknossos/tests/test_annotation.py

```diff
@@ -36,6 +36,58 @@ def test_annotation_from_file() -> None:
         assert len(skeleton_lines) == 32
 
 
+@pytest.mark.vcr()
```

Comment:
I guess that's not needed (anymore), since no download is used?
```suggestion
```

Reply:
You're right. I adapted this from `test_annotation_from_file` where it's probably not needed, either. I remember that I once wrote a test and it took me a while to get it working, since I forgot to add the decorator. So, maybe it would be an easier practice to always use the decorator? However, I see, that it's also a bit unusual to use something when it's not needed. What do you think?

Reply:
cool idea! I just pushed that and removed the decorator everywhere else.